src/main/java/minegame159/meteorclient/modules/movement/Jesus.java

```diff
@@ -113,7 +117,7 @@
 
     private final Setting<Boolean> dipIntoLavaIfFireResistance = sgLava.add(new BoolSetting.Builder()
             .name("dip-into-lava-if-fire-resistance")
-            .description("Lets you go under the lava if you have Fire Resistance effect to avoid fall damage.")
+            .description("Lets you go under the lava if you have Fire Resistance effect to avoid burning.") // rofl someone put "fall damage here"
```

Comment:
Yes, I put here "fall damage" because it literally do this. Because if you will fall to solid liquid over a high height, you get damage:
![solid_liquid_death](https://cdn.discordapp.com/attachments/689197706169942135/799745923348824084/image0.gif)
And "go under lava ... to avoid burning"? That make no sense, doesn't it?

Reply:
Because going under the lava if you have fire resistance is avoiding fall damage?

Reply:
Yes:
![jesus_fall_death](https://user-images.githubusercontent.com/35692622/105611892-41485080-5dc9-11eb-88c5-e5406d222264.gif)
![jesus_fall_safe](https://user-images.githubusercontent.com/35692622/105611894-42797d80-5dc9-11eb-8d8b-1eb243d8c2f8.gif)